### PR TITLE
Merge filters + upstream changes since 8/4

### PR DESCRIPTION
**Pull request by avaleske**
Closing in favor of https://github.com/CityOfDetroit/bloom/pull/358

This needs to be rebased so it doesn't include a bunch of extra commits:
- I initially merged contribute/avaleske-filters into this branch, which included the unsquashed commits for merging filters upstream.
- Now I've merged the squashed commit from upstream as well, so we can rebase it so it doesn't include the other commits

# Pull Request Template

## Issue #293

Addresses #293

## Description

Merge filters work back to main, including upstream changes from 8/4

## How Can This Be Tested/Reviewed?

Please describe the tests that you ran to verify your changes. Provide instructions so we can review. Please also list any relevant details for your test configuration

- [ ] Desktop View
- [ ] Mobile View

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,13 +29,32 @@ All notable changes to this project will be documented in this file. The format
...
+### UI Components
+
+- Added:
+  - Add ResponsiveTable for pricing
+
 ## v1.0.5 08/03/2021
 
 - Added:
-  - Debug flags for public and partner site ([Detroit Team #195](https://github.com/CityOfDetroit/bloom/pull/195), [#1519](https://github.com/bloom-housing/bloom/pull/1519))
+  - Filtering, pagination, and tests for listings endpoint (Parts of Detroit Team [#18](https://github.com/CityOfDetroit/bloom/pull/18), [#133](https://github.com/CityOfDetroit/bloom/pull/133), [#180](https://github.com/CityOfDetroit/bloom/pull/180), [#257](https://github.com/CityOfDetroit/bloom/pull/257), [#264](https://github.com/CityOfDetroit/bloom/pull/264), [#271](https://github.com/CityOfDetroit/bloom/pull/271)) [#1578](https://github.com/CityOfDetroit/bloom/pull/1578)
```
Comment (willrlin) [inline CHANGELOG.md:50]: Should this go up in unreleased? Also, it's re-adding the stuff it clobbered below.

File: backend/core/src/listings/listings.service.ts [hunk 2]
```diff
@@ -69,27 +67,16 @@ export class ListingsService {
...
-    }
-
-    const paginate =
-      // CurrentPage and itemsPerPage are read in from the querystring, so we
-      // confirm the type before proceeding
-      typeof paginationInfo.currentPage === "number" &&
-      paginationInfo.currentPage > 0 &&
-      typeof paginationInfo.itemsPerPage === "number" &&
-      paginationInfo.itemsPerPage > 0
+    // TODO(avaleske): Typesceipt doesn't realize that the `paginate` bool is a
```
Comment (willrlin) [inline backend/core/src/listings/listings.service.ts:70]: type-o 😝

**Review by willrlin — APPROVED** [review]

File: backend/core/src/listings/listings.service.ts (hunk 2 above)
Comment (avaleske) [reply backend/core/src/listings/listings.service.ts:70]: lol gonna fix it upstream

File: CHANGELOG.md (hunk 1 above)
Comment (avaleske) [reply CHANGELOG.md:50]: oops it was in unreleased but git isn't clever

**Comment by avaleske** [comment]
Closing in favor of just using https://github.com/CityOfDetroit/bloom/pull/358, which doesn't have commits in it that were squashed during a different merge.

**Pull request closed without merging**